recipe/meta.yaml

```diff
@@ -88,25 +100,27 @@ requirements:
     # Dec 2020: it seems that git is broken on windows, so we use m2-git
     - patch     # [not win]
     - m2-patch  # [win]
+    - m2-sed    # [win]
     - git       # [not win]
     - m2-git    # [win]
```

Comment:
```suggestion
```
with the switch from git-> tar can we sjust remove all of these?

Reply:
I think we will still need `patch` and `m2-patch` to be able to apply patches. We also still need `m2-sed` as explained in the other comment. However, I think we can remove both `git` and `m2-git`. Let me try that.